src/models/train_model.py

```diff
@@ -24,32 +24,23 @@ def train_classifier(X_train, y_train, penalty='l1', C=1):
 
 
 if __name__ == '__main__':
-	with open(ROOT_DIR / 'data/processed/train.pkl', 'rb') as f:
-		X_train_tfidf, y_train = pickle.load(f)
+    with open(ROOT_DIR / 'data/processed/train.pkl', 'rb') as f:
+        X_train_tfidf, y_train = pickle.load(f)
 
-	# Dictionary of all tags from train corpus with their counts.
-	tags_counts = {}
+    # Dictionary of all tags from train corpus with their counts.
+    tags_counts = {}
 
-	for tags in y_train:
-		for tag in tags:
-			if tag in tags_counts:
-				tags_counts[tag] += 1
-			else:
-				tags_counts[tag] = 1
-
-	mlb = MultiLabelBinarizer(classes=sorted(tags_counts.keys()))
-
-	y_train = mlb.fit_transform(y_train)
-
-	# Store for reuse (at prediction and validation stages)
-	with open(ROOT_DIR / 'models/mlb.pkl', 'wb') as f:
-		pickle.dump(mlb, f)
```

Comment:
Was this deletion intentional?

Reply:
Good catch! No, that should not have been deleted